### PR TITLE
Updating current maintainers: adding @VachaShah, @dblock, @harshavamsi

### PR DESCRIPTION
**Pull request by harshavamsi**
### Description
Cleans up maintainers list to be up to date. Cleans up maintainer and admin responsibilities to track upstream .github.

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._ 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # This should match the team set up in https://github.com/orgs/opensearch-project/teams and include any additional contributors
-*   @opensearch-project/clients @axeoman @deztructor
\ No newline at end of file
+*   @VachaShah @dblock @harshavamsi @axeoman @deztructor @rushiagr @Shephalimittal
```
Comment (axeoman) [inline .github/CODEOWNERS:2]: Weird, `Unknown owner` complains about @harshavamsi and @rushiagr. Could you please double-check the rights?

Comment (dblock) [reply .github/CODEOWNERS:2]: I fixed @harshavamsi by adding him to permissions. I believe @rushiagr should be moved to alumni?

File: ADMINS.md [hunk 2]
```diff
@@ -1,38 +1,21 @@
...
 
-| Admin                    | GitHub ID                                           | Affiliation |
-| -------------------------| ----------------------------------------------------| ----------- |
-| Charlotte                | [CEHENKLE](https://github.com/CEHENKLE)             | Amazon      |
-| Henri Yandell            | [hyandell](https://github.com/hyandell)             | Amazon      |
-| Rushi Agrawal            | [rushiagr](https://github.com/rushiagr)             | Amazon      |
-| Shephali Mittal          | [Shephalimittal](https://github.com/Shephalimittal) | Amazon      |
+| Admin           | GitHub ID                                           | Affiliation |
+| --------------- | --------------------------------------------------- | ----------- |
+| Charlotte       | [CEHENKLE](https://github.com/CEHENKLE)             | Amazon      |
```
Comment (dblock) [inline ADMINS.md:13]: Charlotte is Charlotte Henkle.

I don't think anyone other than her and Henri are actual admins.

File: MAINTAINERS.md [hunk 3]
```diff
@@ -1,76 +1,18 @@
...
-
-Assist, add, and remove [MAINTAINERS](MAINTAINERS.md). Exercise good judgement, and propose high quality contributors to become co-maintainers.
+| Maintainer           | GitHub ID                                           | Affiliation |
+| -------------------- | --------------------------------------------------- | ----------- |
+| Vacha Shah           | [VachaShah](https://github.com/VachaShah)           | Amazon      |
+| Daniel Doubrovkine   | [dblock](https://github.com/dblock)                 | Amazon      |
+| Harsha Vamsi Kalluri | [harshavamsi](https://github.com/harshavamsi)       | Amazon      |
+| Aleksei Atavin       | [axeoman](https://github.com/axeoman)               | Aiven       |
+| Denis Zalevskiy      | [deztructor](https://github.com/deztructor)         | Aiven       |
+| Rushi Agrawal        | [rushiagr](https://github.com/rushiagr)             | External    |
```
Comment (dblock) [inline MAINTAINERS.md:17]: Move to emeritus.

File: ADMINS.md [hunk 4]
```diff
@@ -1,38 +1,21 @@
 - [Overview](#overview)
 - [Current Admins](#current-admins)
 - [Admin Responsibilities](#admin-responsibilities)
-  - [Prioritize Security](#prioritize-security)
-  - [Enforce Code of Conduct](#enforce-code-of-conduct)
-  - [Adopt Organizational Best Practices](#adopt-organizational-best-practices)
 
 ## Overview
 
 This document explains who the admins are (see below), what they do in this repo, and how they should be doing it. If you're interested in becoming a maintainer, see [MAINTAINERS](MAINTAINERS.md). If you're interested in contributing, see [CONTRIBUTING](CONTRIBUTING.md).
```
Comment (VachaShah) [inline ADMINS.md:7]: Overview and Admin Responsibilities section have some duplicate text. Can we make this similar to https://github.com/opensearch-project/OpenSearch/blob/main/ADMINS.md?

**Comment by harshavamsi** [comment]
@dblock @VachaShah updated with asks.

**Review by VachaShah — APPROVED** [review]

**Review by dblock — APPROVED** [review]

**Pull request merged**